src/app/modules/consultations/consultation-profile/consultation-questionnaire/consultation-questionnaire.component.ts

```diff
@@ -187,7 +187,8 @@ export class ConsultationQuestionnaireComponent
   getTranslatedTextForQuestion(item: any) {
     return getTranslatedText(this.currentLanguage, {
       hindi: item?.hindiQuestionText,
-      odia: item?.odiaQuestionText
+      odia: item?.odiaQuestionText,
+      marathi: item?.marathiQuestionText,
```

Comment:
The field names follow the same pattern so hardcoding should not be necessary

Create a function to automatically create such objects

```js
const HINDI = "hindi";
const ODIA = "odia";
const MARATHI = "marathi";

const languages = [HINDI, ODIA, MARATHI];

const createlangObject = ({source, suffix}) => {
  const obj = {};
languages.forEach(lang => {
const key = lang + suffix
const value = source?.[key];
obj[lang] = value;
})

return obj;
}
```

Example usage:
`getLanguageObject({source: item, suffix: "QuestionText"})`